### PR TITLE
Just added a few things

### PR DESCRIPTION
**Pull request by card100**

### CODE REVIEW

File: client/home.html [hunk 1]
```diff
@@ -2,6 +2,27 @@
 <html>
 
 <head>
+	<style>
```
Comment (CosmicWebServices) [inline client/home.html:5]: Not that sure I am liking this style addition. Usually you do not want style unless you really have to when using a library especially when it has a standard for design

File: client/home.html [hunk 2]
```diff
@@ -25,7 +46,7 @@
         <div class="col s12">
             <nav>
                 <div class="nav-wrapper blue">
-                    <a class="brand-logo center" href="/">Cosmic Search Universe</a>
+			<a class="brand-logo center" href="/">Cosmic Search Universe</a> <a class="link" href="/news">News</a>
```
Comment (CosmicWebServices) [inline client/home.html:49]: I think the news search link could have a better place but not here.

**Review by CosmicWebServices — CHANGES_REQUESTED** [review]

**Comment by CosmicWebServices** [comment]
Sorry if I am being too picky I just have a certain design philosophy when it comes to sites.

**Comment by card100** [comment]
I made the <style> because I can't find your css file

**Comment by CosmicWebServices** [comment]
@card100 I have no problem with that I just do not like the styles in it. Also, I am using the materializecss.com library since I am trying to use the material design guidelines to make this.

But theme support are coming soon so you can start making one now if you want.

**Comment by card100** [comment]
Should I then delete this branch?

**Comment by CosmicWebServices** [comment]
Sure

**Pull request closed without merging**